df.world-site.xml

```diff
@@ -431,14 +431,16 @@
             <static-array count='51' pointer-type='site_realization_crossroads'/>
         </static-array>
         <static-array name='river_map' count='51'>
-            <pointer>
-                <static-array name='start' type-name='int32_t' count='48'/>
-                <static-array name='end'   type-name='int32_t' count='48'/>
-                <int32_t name="unk_180"/>
-                <int32_t name="unk_184"/>
-                <int32_t name="start_array_limits"/>
-                <int32_t name="end_array_limits"/>
-            </pointer>
+            <static-array count='51'> 
```

Comment:
Doesn't this generate a 51x51 array? I'm pretty sure that ends up being way larger than what the structure was before, so are you sure this works?